Packs/BreachRx/Integrations/BreachRx/BreachRx.yml

```
@@ -0,0 +1,135 @@
+category: Case Management
+commonfields:
+  id: BreachRx
+  version: -1
+configuration:
+- defaultvalue: https://example.app.breachrx.io
+  display: Your BreachRx URL
+  name: url
+  required: true
+  type: 0
+- name: api_url
+  display: GraphQL API URL
+  required: true
+  defaultvalue: https://graphql.app.breachrx.io
+  type: 0
+- displaypassword: API Key
+  display: API Key
+  additionalinfo: The API Key to use for connection
+  name: api_key
+  required: true
+  hiddenusername: true
+  type: 9
+- displaypassword: Secret Key
+  display: Secret Key
+  additionalinfo: The API Key to use for connection
+  name: secret_key
+  required: true
+  hiddenusername: true
+  type: 9
```

No need for 2 `type: 9` parameters, 1 is enough:

```suggestion
- display: Secret Key
  displaypassword: API Key
  name: credentials
  type: 9
  required: true
  hiddenusername: true
```

Then, in `BreachRx.py`:

```python
api_key = demisto.params().get("credentials", {}).get("password")
secret_key = demisto.params().get("credentials", {}).get("identifier")
```

---

I tried this locally, but now the configuration is only asking for the API key, even though it needs the Secret Key as well to successfully connect.  With this change, how can I provide the secret key?

---

Setting `hiddenusername: false` should expose the username (aka `credentials.identifier`field).

---

This is done.